default.nix

```diff
@@ -47,6 +47,8 @@ let
         pkgs = self.callPackage ({ pkgs }: pkgs) {};
         onLinux = pkg: f: if pkgs.stdenv.isLinux then f pkg else pkg;
       in {
+        shelly = self.callHackage "shelly" "1.9.0" {};
```

Comment:
Might want to upstream this: https://github.com/reflex-frp/reflex-platform/issues/542